CHANGELOG.rst

```diff
@@ -50,6 +50,8 @@ Other
 * Molecule can now be called as a Python module (``python -m molecule``). Patch by `@ssbarnea`_.
 * Add `Travis CI integration`_ and fix related test issues.
 * Add Docker ``buildargs`` option for configuring the ``docker_image`` ``create.yml`` build step.
+* `dependency` step is now run by default before any playbook sequence step, including
```

Comment:
I don't recall if there's automatic changelog creation or not. Should I remove this, throw it into an "unrelease" section above 2.20, or...?

Reply:
Nothing automated yet, yes please, if you could add change log entry, that'd be great :+1:

Reply:
Thanks, done. I left out the "Important Changes"/"Other" subsections since that looked like a special thing for 2.20 and the redhat/ansible adoption. I'm happy to make any desired changes, of course. :)